srml/balances/src/lib.rs

```diff
@@ -54,6 +54,16 @@ pub trait Trait<I: Instance = DefaultInstance>: system::Trait {
 	/// Handler for when a new account is created.
 	type OnNewAccount: OnNewAccount<Self::AccountId>;
 
+	/// Handler for the unbalanced reduction when taking transaction fees.
+	type TransactionPayment: OnUnbalancedDecrease<Self::Balance>;
+
+	/// Handler for the unbalanced reduction when taking fees associated with balance
+	/// transfer (which may also include account creation).
+	type TransferFee: OnUnbalancedDecrease<Self::Balance>;
```

Comment:
There is already storage item `TransferFee`. Better to call it something else to avoid confusion.